cpp/src/Ice/Object.cpp

```diff
@@ -161,11 +161,6 @@ namespace
             case OperationMode::Idempotent:
                 return "::Ice::Idempotent";
         }
-        //
-        // This could not happen with C++11 strong type enums
```

Comment:
Unfortunately GCC doesn't like this change and is complaining.

Reply:
From what I can tell this is the expected GCC behavior. https://gcc.gnu.org/bugzilla/show_bug.cgi?id=87951#c10